tutorials/darcy_and_transport_data.py

```diff
@@ -287,6 +287,10 @@ def assign_data(self, gb, pb, data_class, data_key):
 
         for g, d in gb:
             d[data_key] = data_class(g, d, **pb)
+            
+        for e, d in gb.edges():
+            gh = gb.nodes_of_edge(e)[0]
+            d['kn'] = np.max(gb.node_props(gh)['param'].get_permeability().perm)
 
```

Comment:
@alessiofumagalli I had to add this to make your tutorial run. If you would rather do something else, feel free to change it

Reply:
I have no objections

Reply:
@rbe051 , can you elaborate?

Reply:
At some point we introduces the normal permeability 'kn' explicitly in the code. Before it was just taken from the higher/lower (I don't remember) dimensional grid. The tutorial was probably written for the old code when 'kn' was given explicitly.  Therefore I added the normal permability and just took the maximum permability from the higher dimensional grid as its value. You might want to do something more fance here, especially of there is a non-homogenous permeability